pkg/docker/docker.go

```diff
@@ -563,7 +684,10 @@ func (d *stiDocker) GetAssembleInputFiles(image string) (string, error) {
 }
 
 // getScriptsURL finds a scripts url label in the image metadata
-func getScriptsURL(image *docker.Image) string {
+func getScriptsURL(image *api.Image) string {
+	if image == nil {
+		return ""
+	}
```

Comment:
in the other spots, either the getScriptsURL call preceded it, and essentially shield those methods, or the image struct was constructed before hand, and we just passed the pointer to method vs. the struct itself (save space on the stack, etc.)

that said, I can make the nil check for that as an input parameter consistent, but in part I explicitly didn't do that pervasively because I've been dinged for unnecessary nil checks in prior code reviews (like when the caller insures a param is not nil).

Just let me know which way you want me to go - thx.